packaging/performance-analyzer-agent-cli

```diff
@@ -5,7 +5,7 @@ PA_AGENT_JAVA_OPTS="-Dlog4j.configurationFile=$ES_HOME/plugins/opendistro_perfor
               -XX:InitialBootClassLoaderMetaspaceSize=30720 -XX:MaxRAM=400m"
 
 ES_MAIN_CLASS="com.amazon.opendistro.elasticsearch.performanceanalyzer.PerformanceAnalyzerApp" \
-ES_ADDITIONAL_CLASSPATH_DIRECTORIES=plugins/opendistro_performance_analyzer \
+ES_ADDITIONAL_CLASSPATH_DIRECTORIES=plugins/opendistro_performance_analyzer/lib \
```

Comment:
The agent shouldn't depend on the libs in the plugin's classpath. Should this be
`ES_ADDITIONAL_CLASSPATH_DIRECTORIES=performance_analyzer_rca/lib` instead?

Reply:
libs/ actually are the rca libs (I copied them out)
So you're right, we should be able to avoid the copy and just point to the rca libs directly
I'll add this in the next patch set